modules/deployment/src/test/java/org/mule/runtime/module/deployment/internal/DomainDeploymentWithELServiceTestCase.java

```diff
@@ -84,15 +92,14 @@ public void disposeStaleDomains() {
     TestDomainFactory.after();
   }
 
-  @Override
-  protected File getExpressionLanguageServiceFile(File tempFolder) {
+  private static File getRealExpressionLanguageServiceFile(File tempFolder) {
     final URL weaveJarUrl = WeaveServiceProvider.class.getProtectionDomain().getCodeSource().getLocation();
-    final File defaulServiceSchedulerJarFile = getResourceFile(weaveJarUrl.getFile(), tempFolder);
+    final File defaultServiceSchedulerJarFile = getResourceFile(weaveJarUrl.getFile(), tempFolder);
```

Comment:
We could fix this even further:
```suggestion
    final File defaultServiceJarFile = getResourceFile(weaveJarUrl.getFile(), tempFolder);
```